src/src/AdminBundle/Resources/views/Event/edit.html.twig

```diff
@@ -10,4 +10,87 @@
+        <div class="col-md-6">
+            <p>sem pridu linky na testovanie sablony</p>
+        </div>
+    </div>
+    <div class="row">
+        <div class="col-md-6">
+            <a href="#" data-toggle="modal" data-target="#{{ 'assignModal' ~ form.vars.value.id }}"
+               class='btn btn-primary'>Assign Organizer</a>
+            {% include "@AdminBundle/Resources/views/autocompleteModal.html.twig"
+                with {
+                    modal_id: 'assignModal' ~ form.vars.value.id,
+                    modal_title: 'Assign Organizer',
+                    modal_content: 'Type to assign organizer. ',
+                    action_path: path('event_assign_organizer', {'id':event_type_id, 'event_id': form.vars.value.id}),
+                }
+            %}
+            {% if organizers | length == 0 %}
+                <p>No Organizer found.</p>
+            {% else %}
+                <table class="table table-bordered">
+                    <thead class="thead-light">
+                    <tr>
+                        <th scope="col">#</th>
+                        <th scope="col">Email</th>
+                        <th scope="col">Unassign</th>
+                    </tr>
+                    </thead>
+                    <tbody>
+                    {% for organizer in organizers %}
+                        <tr>
+                            <th scope="row">{{ loop.index }}</th>
+                            <td>{{ organizer.email }}</td>
+                            <td><a href="#" data-toggle="modal" data-target="#deleteModal{{ organizer.id }}">Unassign</a></td>
+                        </tr>
+                        {% include "@AdminBundle/Resources/views/modal.html.twig"
+                            with {
+                                modal_id: 'deleteModal' ~ organizer.id,
+                                modal_title: 'Unassign Organizer',
+                                modal_content: 'Are you sure you want to unassign ' ~ organizer.email ~ ' '~ organizer.id ~'?',
+                                action_path: path('event_unassign_organizer', {
+                                    'id': event_type_id,
+                                    'event_id': form.vars.value.id,
+                                    'organizer_id': organizer.id
+                                    }
+                                ),
+                            }
+                        %}
+                    {% endfor %}
+                    </tbody>
+                </table>
+            {% endif %}
+        </div>
+        <div class="col-md-6"></div>
+    </div>
+{% endblock %}
+
+
+{% block javascripts_footer %}
+    {{ parent() }}
+    <script>
+        $(function() {
+            function log( message ) {
+                $( "<div>" ).text( message ).prependTo( "#log" );
+                $( "#log" ).scrollTop( 0 );
+            }
+
+            $( "#birds" ).autocomplete({
+                source: "{{ path('autocomplete') }}",
```

Comment:
uff.... do JS renderovat twig,... no ano, da sa to, robilo sa to, ale snazime sa kodit slusne, napriek tomu, ze pouzivame **shitty** technologie.

Reply:
No, takto to robili aj v tom druhom tutoriale, nie len v prvom co som nam poslala do skupiny.

Reply:
ano, ako pisem *robilo sa to*, lenze je to nechutne,... ked sa JS presunie do assetov tak to tak ci tak tkato nebude fungovat. path mozes umiestnit napr do formulara alebo nejakeho divu v modali pre autocomplete ako `data-path-url="{{ path('autocomplete') }}"` a tu to len odtial vyberies.

Reply:
nechajme to teda do dalsieho pr, ked upravime js a assety